addons/account/models/account_move.py

```diff
@@ -3655,7 +3655,7 @@ def _compute_cumulated_balance(self):
             return
 
         # get the where clause
-        query = self._where_calc(self.env.context.get('domain_cumulated_balance'))
+        query = self._where_calc(list(self.env.context.get('domain_cumulated_balance')))
```

Comment:
```suggestion
        query = self._where_calc(list(self.env.context.get('domain_cumulated_balance') or []))
```

Reply:
Updated the code. Bette be safe, in case of changes in https://github.com/odoo/odoo/blob/e06aaecf9e8a590604818b09d23fe1de410e6762/addons/account/models/account_move.py#L3649-L3654

Up to this PR the only place where `domain_cumulated_balance` appears is on
https://github.com/odoo/odoo/blob/e06aaecf9e8a590604818b09d23fe1de410e6762/addons/account/models/account_move.py#L3654-L3665